akka-docs/src/main/paradox/testing.md

```diff
@@ -760,13 +759,13 @@ akka {
 
 ## Different Testing Frameworks
 
-Akka’s own test suite is written using [ScalaTest](http://www.scalatest.org),
+Akka’s test suite is written using [ScalaTest](http://www.scalatest.org),
 which also shines through in documentation examples. However, the TestKit and
-its facilities do not depend on that framework, you can essentially use
+its facilities do not depend on that framework, so you can essentially use
 whichever suits your development style best.
 
-This section contains a collection of known gotchas with some other frameworks,
-which is by no means exhaustive and does not imply endorsement or special
+This section contains a collection of known problems with some other frameworks,
```

Comment:
I'm not sure it's nice to characterize these as 'problems with those frameworks', it's more things to be aware of.

```suggestion
This section contains a collection of known gotchas with some other frameworks,
```